public/js/controllers/game.js

```diff
@@ -28,7 +28,6 @@ angular.module('mean.system') //eslint-disable-line
       $scope.getUserInfo = (user, searchForm) => {
         if (searchForm.$dirty) {
```

Comment:
When you are naming variables, use a name that can describe what you want to do or reflect the purpose it will serve. You can also use mnemonics

Reply:
@Orlayhemmy Maybe this [resource](https://scotch.io/tutorials/angularjs-form-validation) can help with your understanding of what $dirty is. Angular provides properties on forms that help us validate them

Reply:
I have checked it out. Thanks